client/src/index.js

```diff
@@ -4,10 +4,13 @@ import 'bootstrap/dist/css/bootstrap.css';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import CardComponent from "../src/cardComponent/cardComponent"
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
+    {/* flickity library used */}
+    <CardComponent/>
```

Comment:
We shouldn't add components directly to `index.js`, components to be added only in `App.js`